frontend/src/Main.purs

```diff
@@ -51,7 +52,10 @@ config state = do
   dt <- extract <$> nowDateTime
 
   pure
-    { initialState: set (socket <<< connection) (Just socket') state
+    {initialState:
+        set (socket <<< connection) (Just socket') $
+        set lang (fromMaybe English $ unsafePerformEff detectLocale)
```

Comment:
@ksaric then lets ask @sectore to fix it. There is no need for unsafePerform here